application/F3DOptionsParser.cxx

```diff
@@ -288,6 +288,7 @@ void ConfigurationOptions::GetOptions(F3DAppOptions& appOptions, f3d::options& o
     this->DeclareOption(grp1, "up", "", "Up direction", options.getAsStringRef("scene.up-direction"), HasDefault::YES, MayHaveConfig::YES, "{-X, +X, -Y, +Y, -Z, +Z}");
     this->DeclareOption(grp1, "axis", "x", "Show axes", options.getAsBoolRef("interactor.axis"), HasDefault::YES, MayHaveConfig::YES);
     this->DeclareOption(grp1, "grid", "g", "Show grid", options.getAsBoolRef("render.grid.enable"), HasDefault::YES, MayHaveConfig::YES);
+    this->DeclareOption(grp1, "grid-absolute", "G", "Show grid at (0,0,0)", options.getAsBoolRef("render.grid.absolute"), HasDefault::YES, MayHaveConfig::YES);
```

Comment:
I'm not a big fan of adding small/big cases for short option. I'd rather not have a short option at all. wdyt @Meakk ?

Reply:
> even just --grid-absolute

Two option to do the same thing with slight difference is a bad idea and will cause headaches in the future.
`-g` turn on the grid, other options configure the grid. Simple (yet verbosy, I know, that is why config files can help :) )

Reply:
I agree, I prefer `-g --grid-absolute`.  
If we want to swap between relative/absolute easily, you could just do `-g --grid-absolute=true` and change `true` by `false`.  
Maybe `0` and `1` is also supported by cxxopts, I have never tried.

Reply:
the council has spoken

Reply:
> you could just do -g --grid-absolute=true and change true by false.
> Maybe 0 and 1 is also supported by cxxopts, I have never tried.

correct, true/false and 0/1 are both supported